### ISSUE TITLE
(Map) Buttons around map are missing focus indicators (49)

### ISSUE THREAD

**Issue by ellinooraikaheimo** [HSLdevcom/jore4#1689]
[Link to slide](https://docs.google.com/presentation/d/1ePFbPmTUUK8m5-11lZZMlUyMNefIVyOrLa_SrK17pCc/edit#slide=id.g1e3be12a1f7_0_110)

Note: Do this after the React map gl is done, cause it is about the map operations.

blocked by: #1662

**Comment by culka** [comment]
Fixing the global theme in #1687 seems fix these.
